ios/Plugin/Plugin.swift

```diff
@@ -295,13 +295,20 @@ public class AdMob: CAPPlugin, GADBannerViewDelegate, GADFullScreenContentDelega
         DispatchQueue.main.async {
             if let rootViewController = UIApplication.shared.keyWindow?.rootViewController {
                 if let ad = self.interstitial {
-                    ad.present(fromRootViewController: rootViewController)
+                    if (ad.isReady) {
```

Comment:
3 level `if`s are not really that good. Can you please check that after each "not ready" you returns?

Reply:
I think so too. I'll work after merged. Thanks.